src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -4,49 +4,101 @@
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List</p>
  */
 public class ArrayList<T> implements List<T> {
+    private final int defaultCapacity = 16;
```

Comment:
@Sasha192 make it as a constant
```suggestion
    private static final int DEFAULT_CAPACITY = 16;
```